cmake/External/aotriton.cmake

```
@@ -4,6 +4,27 @@ if(NOT __AOTRITON_INCLUDED)
   set(__AOTRITON_EXTERN_PREFIX "${CMAKE_CURRENT_BINARY_DIR}/aotriton")
   set(__AOTRITON_INSTALL_DIR "${PROJECT_SOURCE_DIR}/torch")
   add_library(__caffe2_aotriton INTERFACE)
+
+  # AOTriton package information from GitHub Release Pages
+  # Replaces .ci/docker/aotriton_version.txt
+  set(__AOTRITON_VER "0.7b")
+  string(JOIN ";"
+         __AOTRITON_MANYLINUX_LIST
+         "manylinux_2_17"  # rocm6.1
+         "manylinux_2_17"  # rocm6.2
+         )
```

```suggestion
set(__AOTRITON_MANYLINUX_LIST
    "manylinux_2_17"  # rocm6.1
    "manylinux_2_17"  # rocm6.2
   )
```

This can be a little less verbose if the intention is to create a CMake list? Likewise for `__AOTRITON_ROCM_LIST` and `__AOTRITON_SHA256_LIST`

---

If I understand CMake's list correctly, it is is a string with `;` as separator, and the suggested change will make it a string instead of list

---

IIUC from https://cmake.org/cmake/help/latest/command/list.html, `set(var a b c d e) creates a list with a;b;c;d;e`, so specifying the ";" is not mandatory. I tested the suggested change locally and it correctly picked the right value from the list.